frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/InteractiveEmbeddingCTA/InteractiveEmbeddingCTA.tsx

```diff
@@ -53,7 +53,7 @@ export const InteractiveEmbeddingCTA = () => {
       >
         <Text
           align="center"
-          c="white"
+          c="text-swhite"
```

Comment:
```suggestion
          c="text-white"
```
Happens to the best of us 🤪